api/_hyperfunctions/heartbeat_agg/dead_ranges.md

```diff
@@ -0,0 +1,56 @@
+---
+api_name: dead_ranges()
+excerpt: Get the down intervals from a heartbeat_agg
+topics: [hyperfunctions]
+api:
+  license: community
+  type: function
+  toolkit: true
+  experimental: true
+  version:
+    experimental: 1.13.0
+hyperfunction:
+  family: state aggregates
+  type: aggregate
+  aggregates:
+    - heartbeat_agg()
+api_details:
+  summary: |
+    Given a heartbeat aggregate, this will return a set of (start_time,
+    end_time) pairs representing when the underlying system did not have a
+    valid heartbeat during the interval of the aggregate.
+  signatures:
+    - language: sql
+      code: |
+        dead_ranges(
+            agg HEARTBEATAGG
+        ) RETURNS TABLE (
+	        start TIMESTAMPTZ,
+	        end TIMESTAMPTZ
```

Comment:
Check this in the source? Are tabs or something causing this?

Reply:
Oddly there were tabs here, not sure how that happened.